addons/binding/org.openhab.binding.fronius/src/main/java/org/openhab/binding/fronius/internal/FroniusHandlerFactory.java

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.fronius.internal;
+
+import static org.openhab.binding.fronius.FroniusBindingConstants.*;
+
+import org.eclipse.smarthome.core.thing.Bridge;
+import org.eclipse.smarthome.core.thing.Thing;
+import org.eclipse.smarthome.core.thing.ThingTypeUID;
+import org.eclipse.smarthome.core.thing.binding.BaseThingHandlerFactory;
+import org.eclipse.smarthome.core.thing.binding.ThingHandler;
+import org.openhab.binding.fronius.handler.FroniusActiveDeviceInfoHandler;
+import org.openhab.binding.fronius.handler.FroniusInverterRealtimeDataHandler;
+import org.openhab.binding.fronius.handler.FroniusMeterRealtimeDataHandler;
+import org.openhab.binding.fronius.handler.FroniusStorageRealtimeDataHandler;
+import org.openhab.binding.fronius.handler.FroniusSymoBridgeHandler;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The {@link FroniusHandlerFactory} is responsible for creating things and thing
+ * handlers.
+ *
+ * @author Gerrit Beine - Initial contribution
+ */
+public class FroniusHandlerFactory extends BaseThingHandlerFactory {
```

Comment:
Is it correct no OSGI service is generated for this file? (I see gitignore in OSGI-INF map but no `Component` annotations, among others that I would expect on this class)